src/front/styles/pattern-library.scss

```diff
@@ -564,3 +564,24 @@ hr.hr-strong {
         list-style-position:inside;
     }
 }
+
+.pl-pattern {
+    .fonts img {
+       float: right;
+       padding-left: 20px;
+       width: 500px;
+       max-width: 500px;
+       height: 400px;
+    }
+}
+
+.pl-pattern {
+    #myModalLive .modal-dialog {
+        margin: 30% auto;
+        max-width: 90%; 
+    }
+}
+
+.filter-spacer {
```

Comment:
Maybe we prefix pl to this to avoid confusion?

Reply:
good idea, this is done.

Reply:
ok, all requested updates should be done.